pkgs/by-name/ka/kanboard/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nixosTests,
+  nix-update-script,
+  php,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "kanboard";
+  version = "1.2.42";
+
+  src = fetchFromGitHub {
+    owner = "kanboard";
+    repo = "kanboard";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-/Unxl9Vh9pEWjO89sSviGGPFzUwxdb1mbOfpTFTyRL0=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/kanboard
+    cp -rv . $out/share/kanboard
+
+    runHook postInstall
+  '';
+
+  passthru = {
+    updateScript = nix-update-script { };
+    tests = lib.optionalAttrs stdenvNoCC.hostPlatform.isLinux {
+      inherit (nixosTests) kanbaord;
```

```suggestion
      inherit (nixosTests) kanboard;
```
Wait...almost missed this. I'm surprised OfBorg continues with this error

---

😢 has been fixed